DataLayer/CmuData/CmuData.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <QList>
+
+#include "I_CmuUnit.h"
+
+class CmuData: I_CmuData
+{
+public:
+    CmuData(const unsigned char& numberOfCells);
+
+    I_CmuUnit getCmuUnit(const unsigned char& index);
+
+private: 
+    QList<I_CmuUnit*> cmuUnits_;
```

Comment:
Implemented it using `QScopedArrayPointer` before seeing response, will change it if the current implementation doesn't work